### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by Frityet** [teal-language/tl#725] [Community]
Is it possible to donate to this project, or to hishamn himself? I really like this and would like to help out a bit.

**Comment by hishamhm** [comment]
_Right now_ I don't think I'm in a position to take donations, given how irregularly I'm working on this project (sometimes big bursts of activity, sometimes I let it sit for a while... but slowly but surely making progress). But thank you for the sentiment, it's most appreciated!

If the situation changes in the future, I'll make sure to post about it here!
